src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -137,13 +137,28 @@ public List<Island> getTeams() {
     @Override
     public CompletableFuture<Island> createTeam(@NotNull Player owner, String name) {
         CompletableFuture<String> schematicNameCompletableFuture = new CompletableFuture<>();
-        owner.openInventory(new CreateGUI(owner.getOpenInventory().getTopInventory(), schematicNameCompletableFuture).getInventory());
+
+        if(IridiumSkyblock.getInstance().getSchematics().schematics.entrySet().size() > 1) {
+            owner.openInventory(new CreateGUI(owner.getOpenInventory().getTopInventory(), schematicNameCompletableFuture).getInventory());
+        }
+
         return CompletableFuture.supplyAsync(() -> {
-            String schematic = schematicNameCompletableFuture.join();
-            if (schematic == null) return null;
+
+            String schematic = null;
+
+            if(IridiumSkyblock.getInstance().getSchematics().schematics.entrySet().size() == 1) {
+                for (Map.Entry<String, Schematics.SchematicConfig> entry : IridiumSkyblock.getInstance().getSchematics().schematics.entrySet()) {
+                    schematic = (entry.getKey());
+                }
+            } else {
+                schematic = schematicNameCompletableFuture.join();
+            }
+
+            String finalSchematic = schematic;
+            if (finalSchematic == null) return null;
 
             User user = IridiumSkyblock.getInstance().getUserManager().getUser(owner);
-            Schematics.SchematicConfig schematicConfig = IridiumSkyblock.getInstance().getSchematics().schematics.get(schematic);
+            Schematics.SchematicConfig schematicConfig = IridiumSkyblock.getInstance().getSchematics().schematics.get(finalSchematic);
```

Comment:
Would moving this to a new method make more sence?

private CompletableFuture<String> getSchematic(Player owner);

Reply:
it would probably enhance code readability, but as far as functionality goes it seems to be working fine
if we want to we can do that